### DISCUSSION TITLE
How to solve this? The relative path of the object does not change./これを解決するには?オブジェクトの相対パスは変わりません。

### DISCUSSION THREAD

**Discussion by MowChanlol** [bdunderscore/modular-avatar#630] [Q&A]
![2b0e6012192941978c6bb98f6cb0c2b7](https://github.com/bdunderscore/modular-avatar/assets/157624108/9ca439ad-a013-4744-bc7d-77ee25300d7b)
![6ff685de898a435f4523fca722c871e7](https://github.com/bdunderscore/modular-avatar/assets/157624108/ac20529d-98a8-42cb-a7c6-8123cf4a34b3)
I want to apply an action to this Loli_Chiffon_kaihen, but I don't know how to solve it.
このloli _ chiffon _kaihenにアクションを適用したいのですが、解決方法がわかりません。

**Comment by bdunderscore** [comment]
Hi - the question is a bit vague. Can you please describe:

- What you are trying to accomplish
- How you were trying to accomplish it
- Which part of the process you were stuck on, or what strange behavior you encountered

**Comment by MowChanlol** [comment]
I put an avatar B inside my avatar A. I try to make an animation of avatar B. I used MA Morge Animator and its Path Mode:Reiatice to this object does not move Animator. You can try this operation yourself. But I have reached my idea in another way, so you don't have to spend too much time helping me.
![S%8 XL9BAC${YVF{FNH)(P5](https://github.com/bdunderscore/modular-avatar/assets/157624108/5081b434-d75c-417e-817e-894fb9773e57)
![0QYTR3KMK)3W`D)~94EMENU](https://github.com/bdunderscore/modular-avatar/assets/157624108/52e7a349-c304-4936-93fd-96159873b575)

**Comment by bdunderscore** [comment]
Attaching an animator won't work, unfortunately; you'll have to build an animator controller for the action layer. MA can help by merging that controller in (using MA Merge Animator), but the controller itself has to be build to the usual VRChat action layer spec in order to be able to turn it on/off. This includes enabling the Action playable layer when the animations need to play.
